diode-server/reconciler/ops.go

```diff
@@ -127,6 +129,6 @@ func handleChangeSetError(err error) error {
 	return &changeset.Error{
 		Message: err.Error(),
 		Code:    diodeErrors.ErrCodeInternal,
-		Details: []byte{},
+		Details: []byte(genericInternalServerErrorDetails),
 	}
```

Comment:
empty bytes fails json.Marshal in the repository.  we could allow that in the repository additionally, but I just put a generic error message in this particular case.  can be whatever.

Reply:
If we prefer no details for this `[]byte(`null`)` also works here ...

Reply:
Adding `omitempty` into https://github.com/netboxlabs/diode/blob/feat-postgres-implementation-and-enhancements/diode-server/reconciler/changeset/changeset.go#L69 would solve empty bytes issue I believe

Reply:
sure, added.